packaging/hudi-spark-bundle/pom.xml

```diff
@@ -140,14 +140,14 @@
                 <!-- NOTE: We have to relocate all classes w/in org.apache.spark.sql.avro to avoid
                            potential classpath collisions in case users would like to also use "spark-avro" w/in
                            their runtime, since Hudi carries some of the same classes as "spark-avro" -->
-                <relocation>
-                  <pattern>javax.servlet.</pattern>
-                  <shadedPattern>org.apache.hudi.javax.servlet.</shadedPattern>
-                </relocation>
                 <relocation>
                   <pattern>org.apache.spark.sql.avro.</pattern>
                   <shadedPattern>org.apache.hudi.org.apache.spark.sql.avro.</shadedPattern>
                 </relocation>
+                <relocation>
+                  <pattern>javax.servlet.</pattern>
+                  <shadedPattern>org.apache.hudi.javax.servlet.</shadedPattern>
+                </relocation>
```

Comment:
unnecessary moving of these lines

Reply:
These lines were inserted b/w the ones above and the comment attached to them, messing it up